Ginger/GingerCoreNET/Database/ActDBValidation.cs

```diff
@@ -147,7 +147,7 @@ public string SQL
         public ObservableList<ActInputValue> QueryParams = new ObservableList<ActInputValue>();
 
         [IsSerializedForLocalRepository]
-        public ObservableList<CosmosPatchInputValues> CosmosPatchInputValues = new ObservableList<CosmosPatchInputValues>();
+        public ObservableList<ActInputValue> CosmosPatchInputValues = new ObservableList<ActInputValue>();
```

Comment:
IsSerializedForLocalRepository is not requeired as act input values are are already being serialized in base class.